extensions/amp-audio/1.0/base-element.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Audio} from './component';
+import {EMPTY_METADATA} from '../../../src/mediasession-helper';
+import {PreactBaseElement} from '#preact/base-element';
+
+export class BaseElement extends PreactBaseElement {}
+
+/** @override */
+BaseElement['Component'] = Audio;
+
+/** @override */
+BaseElement['props'] = {
+  album: {attr: 'album', type: 'string', default: EMPTY_METADATA.album},
```

Comment:
I'd recommend applying the default `EMPTY_METADATA` information (and other defaults for that matter) at the level of the Preact layer, so that it is included only once and available in all environments. Something like the following in the props destructure:
```js
// component.js
const {
  album = EMPTY_METADATA.album, 
  artist = EMPTY_METADATA.artist, 
  loop = false,
  ...
} = props 
```